mitodl/hugo-course-publisher
# Mobile search filter mode

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #224 

#### What's this PR do?
Modifies the facet display for narrower browser widths.

#### How should this be manually tested?
View the search page at different widths and in mobile mode, make sure it looks and behaves as expected (similar to open course search).

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)

##### Browser < 1000px wide:
<img width="956" alt="Screen Shot 2020-10-07 at 8 48 12 AM" src="https://user-images.githubusercontent.com/187676/95333624-03a3d400-087b-11eb-9907-c081011e0c29.png">

<img width="966" alt="Screen Shot 2020-10-07 at 8 48 21 AM" src="https://user-images.githubusercontent.com/187676/95333639-09011e80-087b-11eb-96d7-2de4943b2c7a.png">


##### Mobile view:
<img width="412" alt="Screen Shot 2020-10-07 at 8 48 40 AM" src="https://user-images.githubusercontent.com/187676/95333674-13bbb380-087b-11eb-93dd-f052fb244c9f.png">

<img width="412" alt="Screen Shot 2020-10-07 at 8 48 47 AM" src="https://user-images.githubusercontent.com/187676/95333686-174f3a80-087b-11eb-97b4-c107bbd1dd24.png">
